### ISSUE TITLE
The inferred locality is Santa Catalina Island

### ISSUE THREAD

**Issue by jhnwllr** [gbif/portal-feedback#5213]
(from helpdesk) 

Feedback from GBIF.org regarding: https://www.gbif.org/occurrence/4110426256
Occurrence Detail 4110426256
Global Biodiversity Information Facility. Free and Open Access to Biodiversity Data.
www.gbif.org

It appears this record is mis-mapped —as noted in the "Occurrence remarks," the inferred locality is Santa Catalina Island.

**Comment by jhnwllr** [comment]
(user cc'ed publisher)

**Comment by jhnwllr** [comment]
The record has a huge coordinateUncertainty and appears to be a centroid of CA. 

![image](https://github.com/gbif/portal-feedback/assets/4245213/13ad852d-b104-482f-9c2b-29d04fd07e57)

**Comment by sformel-usgs** [comment]
I agree, it's not quite clear how Catalina was inferred.  I have replied to the original email to see if the curator has any additional information.
